tests/acceptance/guid-registration/forks-test.ts

```diff
@@ -12,6 +12,10 @@ module('Acceptance | guid-registration/forks', hooks => {
     setupOSFApplicationTest(hooks);
     setupMirage(hooks);
 
+    hooks.beforeEach(() => {
+        server.create('registration-provider', { id: 'osf' });
```

Comment:
@aaxelb Seems like most (if not all) of the test failures are related to needing an `osf` registration provider created. Would you suggest that I go through each failing test and create the factory like this? Or I could go down the path of doing some sort of global beforeEach and creating it there? Or perhaps you know of another way?

Reply:
updating the failing tests seems better than a global beforeEach, i think

but rather than creating the provider explicitly every time, you could update the [registration factory](https://github.com/robustdj/ember-osf-web/blob/19cc7fcc51a444c0cebab13475bcc2cd0f833a6d/mirage/factories/registration.ts#L117) to create the `osf` provider if it doesn't already exist -- then you'll only have to update tests that go to a registries route without adding a registration